tocount/functions.py

```diff
@@ -16,16 +16,29 @@ class _TextEstimatorRuleBased(Enum):
     DEFAULT = UNIVERSAL
 
 
+class _TextEstimatorTikTokenR50K(Enum):
+    """TikToken R50K text token estimator enum."""
+
+    LINEAR_ALL = "TIKTOKEN R50K LINEAR ALL"
+    LINEAR_ENGLISH = "TIKTOKEN R50K LINEAR ENGLISH"
+    DEFAULT = LINEAR_ENGLISH
```

Comment:
capital case space separated is a bit uncommon; not a major comment but consider changing it to something more suitable if you have other change requests.